build-macos/CalChart.xcodeproj/project.pbxproj

```diff
@@ -127,15 +127,17 @@
 		C7C5FBD81479B385004E5C5B /* OpenGL.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = C7C5FBD71479B385004E5C5B /* OpenGL.framework */; };
 		C7C5FBDC1479B3F8004E5C5B /* WebKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = C7C5FBDB1479B3F8004E5C5B /* WebKit.framework */; };
 		C7D4E2C411D087CD007D3587 /* calchartapp.cpp in Sources */ = {isa = PBXBuildFile; fileRef = C7D4E2C211D087CD007D3587 /* calchartapp.cpp */; };
-		C7E047C8155F279000D8C71B /* field_view.cpp in Sources */ = {isa = PBXBuildFile; fileRef = C7E047C6155F279000D8C71B /* field_view.cpp */; };
-		C7E047CC155F7B8A00D8C71B /* field_canvas.cpp in Sources */ = {isa = PBXBuildFile; fileRef = C7E047CA155F7B8900D8C71B /* field_canvas.cpp */; };
+		C7E047C8155F279000D8C71B /* FieldView.cpp in Sources */ = {isa = PBXBuildFile; fileRef = C7E047C6155F279000D8C71B /* FieldView.cpp */; };
+		C7E047CC155F7B8A00D8C71B /* FieldCanvas.cpp in Sources */ = {isa = PBXBuildFile; fileRef = C7E047CA155F7B8900D8C71B /* FieldCanvas.cpp */; };
 		C7E047CF155F892500D8C71B /* field_frame.cpp in Sources */ = {isa = PBXBuildFile; fileRef = C7E047CD155F892400D8C71B /* field_frame.cpp */; };
 		C7E047D215609A1600D8C71B /* setup_wizards.cpp in Sources */ = {isa = PBXBuildFile; fileRef = C7E047D015609A1200D8C71B /* setup_wizards.cpp */; };
 		C7E8DED3117B6A4C00C9FAC9 /* calchart.icns in Resources */ = {isa = PBXBuildFile; fileRef = C7E8DED2117B6A4C00C9FAC9 /* calchart.icns */; };
 		E807BD5F17ED71F60052A53F /* main.cpp in Sources */ = {isa = PBXBuildFile; fileRef = E807BD5E17ED71F60052A53F /* main.cpp */; };
 		E807BD6117ED71F60052A53F /* calchart_cmd.1 in CopyFiles */ = {isa = PBXBuildFile; fileRef = E807BD6017ED71F60052A53F /* calchart_cmd.1 */; };
+		E80C010222639F1F008D8330 /* ContinuityEditorPopup.h in Headers */ = {isa = PBXBuildFile; fileRef = E80C010022639F1F008D8330 /* ContinuityEditorPopup.h */; };
```

Comment:
I think the `ContinuityEditorPopup` files may not have been committed?

Reply:
oops.  Let me check

Reply:
Oops, fixed.